.pre-commit-config.yaml

```diff
@@ -0,0 +1,47 @@
+default_stages:
+- commit
+repos:
+# general hooks to verify or beautify code
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v3.3.0
+  hooks:
+  - id: check-added-large-files
+    args: [--maxkb=5000]
+  - id: trailing-whitespace
+  - id: check-json
+  - id: check-merge-conflict
+  - id: check-xml
+  - id: check-yaml
+  - id: detect-private-key
+  - id: mixed-line-ending
+  - id: pretty-format-json
+    args: [--autofix]
+
+
+# autoformat code with black formatter
+- repo: https://github.com/psf/black
+  rev: 22.3.0
+  hooks:
+  - id: black
+    args: [--check]
+
+# beautify and sort imports
+- repo: https://github.com/pycqa/isort
+  rev: 5.12.0
+  hooks:
+  - id: isort
+
+
+# check code style
+- repo: https://github.com/pycqa/flake8
+  rev: 3.8.4
+  hooks:
+  - id: flake8
+
+
+# static type checking
```

Comment:
Could you please add a link to jira task describing why is this commented or write it here?

Reply:
Yes: I added a comment and TODO in the code.